stdlib/public/Concurrency/Task.swift

```diff
@@ -1030,6 +1029,17 @@ internal func _asyncMainDrainQueue() -> Never
 @_silgen_name("swift_task_getMainExecutor")
 internal func _getMainExecutor() -> Builtin.Executor
 
+@available(SwiftStdlib 5.1, *)
+@_extern(c, "exit")
+@usableFromInline
+internal func exit(_: CInt) -> Never
+
+@available(SwiftStdlib 5.11, *)
+@_alwaysEmitIntoClient
+internal func _swiftExit(_ exitCode: CInt) -> Never {
```

Comment:
What is the `_swiftExit` for. (could the concurrency intrinsic lookup do a lookup for the `exit` function directly instead?)